packages/@aws-cdk/aws-codebuild/lib/artifacts.ts

```diff
@@ -1,11 +1,27 @@
 import s3 = require('@aws-cdk/aws-s3');
+import { Construct } from '@aws-cdk/cdk';
 import { CfnProject } from './codebuild.generated';
-import { Project } from './project';
+import { IProject } from './project';
+
+/**
+ * The type returned from {@link IArtifacts#bind}.
+ */
+export interface ArtifactsConfig {
+  readonly artifactsProperty: CfnProject.ArtifactsProperty;
+}
+
+export interface IArtifacts {
+  readonly identifier?: string;
```

Comment:
why is identifier needed?

Reply:
It's used for naming your sources/artifacts, and is required when using secondary sources/artifacts.